common/src/main/resources/data/tardis_refined/dimension_type/tardis.json

```diff
@@ -5,8 +5,8 @@
   "respawn_anchor_works": true,
   "bed_works": true,
   "has_raids": false,
-  "has_skylight": false,
-  "has_ceiling": false,
+  "has_skylight": true,
```

Comment:
What is the benefit of a sky light?

Reply:
Remnant from experimentation. I've reverted it.